wpv.sh

```diff
@@ -52,8 +53,9 @@ function trailingslashit {
 SRC_DIR=$(pwd)
 DEST_DIR=""
 
-while getopts ":cs:d:hn" argname; do
+while getopts "cp:s:d:hn" argname; do
```

Comment:
The getopts string must start with `:` so that getopts operates in "silent" mode.

```suggestion
while getopts ":cp:s:d:hn" argname; do
```

Reply:
Fixed. Thanks for the suggestion :D